### PR TITLE
Unity dockerfile

### PR DESCRIPTION
**Pull request by holycattle**
review me @takayukioda 
fyi @shunjikonishi

### CODE REVIEW

File: docker/all/Dockerfile [hunk 1]
```diff
@@ -31,3 +31,14 @@ RUN git clone https://github.com/skohar/cache-for-scala-sbt.git
...
+
+# install Unity
+RUN wget --quiet http://download.unity3d.com/download_unity/unity-editor-5.1.0f3+2015091501_amd64.deb
+#install gdebi to handle dependencies for you
+RUN apt-get install -y gdebi
+RUN gdebi /root/unity-editor-5.1.0f3+2015091501_amd64.deb
+RUN apt-get install xvfb
+RUN apt-get install pulseaudio
+RUN echo '#!/bin/bash' >> /usr/bin/xvfb-unity
+RUN echo 'xvfb-run /opt/Unity/Editor/Unity -projectPath $1 -batchmode -nographics -executeMethod $2 -resultFilePath=$3 -logFile'
```
Comment (takayukioda) [inline docker/all/Dockerfile]: Isn't this should redirect to /usr/bin/xvfb-unity ?

**Comment by takayukioda** [comment]
@holycattle fix me where I noted.
BTW, we will remove this part after I ansiblized it.

**Comment by holycattle** [comment]
Sorry, my bad. Fixed it already. Do you want me to convert this to Ansible as well?

pls review me @takayukioda

File: docker/all/Dockerfile [hunk 2]
```diff
@@ -31,3 +31,14 @@ RUN git clone https://github.com/skohar/cache-for-scala-sbt.git
...
+
+# install Unity
+RUN wget --quiet http://download.unity3d.com/download_unity/unity-editor-5.1.0f3+2015091501_amd64.deb
+#install gdebi to handle dependencies for you
+RUN apt-get install -y gdebi
+RUN gdebi /root/unity-editor-5.1.0f3+2015091501_amd64.deb
+RUN apt-get install -y xvfb
+RUN apt-get install -y pulseaudio
+RUN echo '#!/bin/bash' >> /usr/bin/xvfb-unity
+RUN echo '/usr/bin/xvfb-unity /opt/Unity/Editor/Unity -projectPath $1 -batchmode -nographics -executeMethod $2 -resultFilePath=$3 -logFile'
```
Comment (takayukioda) [inline docker/all/Dockerfile]: I think 

``` Dockerfile
RUN echo '/usr/bin/xvfb-unity /opt/Unity/Editor/Unity -projectPath $1 -batchmode -nographics -executeMethod $2 -resultFilePath=$3 -logFile'
```

should be

``` Dockerfile
RUN echo '/usr/bin/xvfb-unity /opt/Unity/Editor/Unity -projectPath $1 -batchmode -nographics -executeMethod $2 -resultFilePath=$3 -logFile' >>/usr/bin/xvfb-unity
```

isn't it?

**Comment by takayukioda** [comment]
Other than where I noted, LGTM

**Comment by takayukioda** [comment]
> Sorry, my bad. Fixed it already. Do you want me to convert this to Ansible as well

Sorry, I totally forgot to reply this comment :(
Yes, I would be very happy if it's possible :) . please make a new role for unity. Let me know if you have any question about ansible ;)

**Pull request closed without merging**